testing/ef_tests/src/handler.rs

```diff
@@ -84,15 +106,56 @@ macro_rules! bls_handler {
                 "bls"
             }
 
+            fn config_name() -> &'static str {
+                "bls12-381-tests"
+            }
+
             fn handler_name(&self) -> String {
                 $handler_name.into()
             }
+
+            fn run(&self) {
+                let fork_name = ForkName::Base;
+                let fork_name_str = fork_name.to_string();
+                let handler_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+                    .join("consensus-spec-tests")
+                    .join(Self::config_name())
+                    .join(self.handler_name());
+
+                let as_file = |entry: Result<DirEntry, std::io::Error>| -> Option<DirEntry> {
+                    entry
+                        .ok()
+                        .filter(|e| e.file_type().map(|ty| ty.is_file()).unwrap_or(false))
```

Comment:
```suggestion
                        .filter(|e| e.file_type().map(|ty| ty.is_file()).unwrap())
```

I thought this may follow the "fail loudly" ethos more closely? My understanding is that this should never fail unless there's a filesystem/IO error, but perhaps I've missed something?

Reply:
I think I forgot because it was long ago. But I think it is no harm to fail loudly here.

Reply:
I don't think this got implemented but I'm OK with that because any missed files should be detected via the "all files checked" script.